xds/internal/xdsclient/tests/cds_watchers_test.go

```diff
@@ -103,7 +103,7 @@ func verifyClusterUpdate(ctx context.Context, updateCh *testutils.Channel, wantU
 			return fmt.Errorf("received update with error type %v, want %v", gotType, wantType)
 		}
 	}
-	cmpOpts := []cmp.Option{cmpopts.EquateEmpty(), cmpopts.IgnoreFields(xdsresource.ClusterUpdate{}, "Raw", "LBPolicy")}
+	cmpOpts := []cmp.Option{cmpopts.EquateEmpty(), cmpopts.IgnoreFields(xdsresource.ClusterUpdate{}, "Raw", "LBPolicy", "TelemetryLabels")}
```

Comment:
Why is it that in some tests, you include the unknown CSM labels in the `want` section, but in some, you chose to ignore that field?

Reply:
The tests I ignore them in (which I believe were written by you) seem to be testing the logic of the xDS Chain of watches. So the only thing verified are the watch names. So for those tests, I didn't feel as if it was appropriate to test this emission (for the other segments of the struct were also being ignored).